### PR TITLE
#1343 Added autodel of samba credentials on user delete - tested and working

### PR DESCRIPTION
**Pull request by MFlyer**
Hi @schakrava , have nothing to tell - subject says all :wink: 

Important: PR #1336 before this - working on same files

### CODE REVIEW

File: src/rockstor/system/users.py [hunk 1]
```diff
@@ -100,6 +100,9 @@ def userdel(uname):
     except KeyError:
         # user doesn't exist
         return
+    #Ensure user get deleted from samba pass db
+    cmd = [SMBPASSWD, '-x', uname]
+    p = subprocess.Popen(cmd, shell=False, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```
Comment (schakrava) [inline src/rockstor/system/users.py]: Hi @MFlyer . Why not use run_command utility method instead of calling subprocess.Popen?

**Comment by MFlyer** [comment]
Hi @schakrava , your right about run_command! Done over 4285515 : no need of try except because all managed users have a samba entry so every delete action will have a return code of 0 :wink: 

Flyer

**Comment by schakrava** [comment]
Thanks @MFlyer, nicely done as always!

**Comment by MFlyer** [comment]
Hi @schakrava , it was a pleasure and had fun as always! :)

**Pull request merged**